gammapy/obs/data_store.py

```diff
@@ -99,6 +99,25 @@ def from_name(cls, name):
         dm = DataManager()
         return dm[name]
 
+    @classmethod
```

Comment:
I need this for the spectrum analysis class. Usually on would pass a name, but for testing purposes the gammapy-extra repo has to be used